packages/xo-server/config.toml

```diff
@@ -109,6 +109,7 @@ offlineSnapshot = false
 snapshotRetention = 0
 timeout = 0
 vmTimeout = 0
+unconditionalSnapshot = false
```

Comment:
Due to current architecture, should be added in `@xen-orchestra/proxy/config.toml`.